test/parallel/test-http-chunked-304.js

```diff
@@ -12,32 +12,33 @@ var net = require('net');
 // Likewise for 304 responses. Verify that no empty chunk is sent when
 // the user explicitly sets a Transfer-Encoding header.
 
-test(204, function() {
+test(204, () => {
   test(304);
 });
 
 function test(statusCode, next) {
-  var server = http.createServer(function(req, res) {
+  const server = http.createServer(common.mustCall((req, res) => {
     res.writeHead(statusCode, { 'Transfer-Encoding': 'chunked' });
     res.end();
     server.close();
-  });
+  }));
 
-  server.listen(0, function() {
-    var conn = net.createConnection(this.address().port, function() {
-      conn.write('GET / HTTP/1.1\r\n\r\n');
+  server.listen(0, common.mustCall(function() {
+    const conn = net.createConnection(this.address().port,
+      common.mustCall(() => {
+        conn.write('GET / HTTP/1.1\r\n\r\n');
 
-      var resp = '';
-      conn.setEncoding('utf8');
-      conn.on('data', function(data) {
-        resp += data;
-      });
+        let resp = '';
+        conn.setEncoding('utf8');
+        conn.on('data', (data) => {
+          resp += data;
+        });
 
-      conn.on('end', common.mustCall(function() {
-        assert.equal(/^Connection: close\r\n$/m.test(resp), true);
-        assert.equal(/^0\r\n$/m.test(resp), false);
-        if (next) process.nextTick(next);
+        conn.on('end', common.mustCall(function() {
+          assert.strictEqual(/^Connection: close\r\n$/m.test(resp), true);
+          assert.strictEqual(/^0\r\n$/m.test(resp), false);
```

Comment:
@thefourtheye as I see it, this is testing this code: https://github.com/nodejs/node/blob/master/lib/_http_outgoing.js#L255-L279